.ado/jobs/cli-init.yml

```diff
@@ -76,7 +76,7 @@ parameters:
           - Name: ARM64ReleaseCsHermes
             language: cs
             configuration: Release
-            platform: arm64
+            platform: ARM64
```

Comment:
Someone recently made them all lowercase...

Reply:
Yes. There was an inconsistency with some lowercase and some upper that showed up in CI, but not PR. I ended up converting them to lowercase to match a CI template, but VS-related bits are all upper case. Changed everything here to all upper case instead.